turbinia/pubsub.py

```diff
@@ -60,20 +60,22 @@ def setup(self):
   def setup_publisher(self):
     """Set up the pubsub publisher."""
     config.LoadConfig()
-    self.publisher = pubsub.PublisherClient()
+    self.publisher = gcp_common.CreateService('pubsub', 'v1')
     self.topic_path = self.publisher.topic_path(
         config.TURBINIA_PROJECT, self.topic_name)
     try:
       log.debug('Trying to create pubsub topic {0:s}'.format(self.topic_path))
-      self.publisher.create_topic(self.topic_path)
+      topics_client = self.publisher.topics()
+      gcp_common.ExecuteRequest(
+          topics_client, 'create', {'name': self.topic_path})
     except exceptions.Conflict:
       log.debug('PubSub topic {0:s} already exists.'.format(self.topic_path))
```

Comment:
I think this should work if you import `import googleapiclient.http` and `from turbinia import TurbiniaException`
```suggestion
    except HttpError as exception:
      if exception.resp.status == 409:
        log.debug('PubSub topic {0:s} already exists.'.format(self.topic_path))
      raise TurbiniaException(
          'Unknown error occurred when creating Topic:'
          ' {0!s}'.format(exception), __name__) from exception
```